client/src/components/RatingsListEntry.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const RatingsListEntry = (props) => {
+  console.log(props, 'THIS IS props')
+  const checkIn = props.checkIn.reduce((acc, cur) => acc + cur, 0);
+  const communication = props.communication.reduce((acc, cur) => acc + cur, 0);
+  const location = props.location.reduce((acc, cur) => acc + cur, 0);
+  const accuracy = props.accuracy.reduce((acc, cur) => acc + cur, 0);
+  const cleanliness = props.cleanliness.reduce((acc, cur) => acc + cur, 0);
+  const value = props.value.reduce((acc, cur) => acc + cur, 0);
+  const rating = (num) => Number.parseFloat(num / 100).toFixed(1);
```

Comment:
This section is neatly done! I like the combo of reduce and a reusable rating function

Reply:
Thank you!